operator/bundle_helpers/patch-csv.py

```diff
@@ -170,10 +170,8 @@ def parse_args():
                         help='The first version of the operator that was published')
     parser.add_argument("--operator-image", required=True, metavar='image',
                         help='Which operator image to use in the patched CSV')
-    parser.add_argument("--no-related-images", action='store_true',
-                        help='Disable passthrough of RELATED_IMAGE_* environment variables')
-    parser.add_argument("--inject-related-images", action='store_true',
-                        help='Create spec.relatedImages list, based on RELATED_IMAGE_* variables in the current environment.')
+    parser.add_argument("--related-images-mode", choices=["downstream", "omit", "konflux"], default="downstream",
+                        help="Set mode of operation for handling related image settings.")
```

Comment:
1. [nit] When I call `patch-csv.py --help`, it does not tell that `donwstream` is default. Could you please add that mention?
2. Could you please add a brief explanation what each mode does w.r.t. RELATED_IMAGE_ env variables and `spec.relatedImages`? This will help users and code readers grasp the idea without having to dive in the code.

Reply:
done